crates/Cargo.toml

```diff
@@ -7,6 +7,19 @@ members = [
 ]
 resolver = "2"
 
+[workspace.package]
+version = "0.7.0"
```

Comment:
I'm expecting these package versions to start to diverge (at least for `flipperzero-sys`), but this is fine for now.

Reply:
I think that it's fine to leave it with workspace default as for now and replace in specific crates on demand: for example, most may use workspace's `A.B.C` by using `version.workspace = true` while auto-generated crate would stick to `flipperzero-firmware` version by explicitly specifying it via `version = 'D.E.F-b.G`.